tests/test_roads.py

```diff
@@ -7,32 +7,25 @@
 
 import os
 
+import arcpy
 import pytest
 from arcgisscripting import ExecuteError  # pylint: disable=no-name-in-module
 
-import arcpy
-
 table_name = 'Roads_Edit'
-sde = os.path.join(os.path.dirname(__file__), '..', 'pro-project', 'localhost.sde')
+sde = os.path.join(os.path.dirname(__file__), '..', 'pro-project', 'localhost@utrans.sde')
 TABLE = os.path.join(sde, table_name)
+TEST_ATTRIBUTE = 'UTRANS_NOTES'
 
 # pragma pylint: disable=no-member
 
-wkt = (
-    'POLYGON ((419771.30999999959 4476140.9299999997, 421271.66000000015 4474303, 425713.20999999996 4477928.7599999998, 424212.86000000034 '
-    '4479766.6799999997, 419771.30999999959 4476140.9299999997)), 26912)'
-)
-
 rules = [
-    'rule.names',
+    'FULLNAME',
 ]
 
 
 def cleanup():
     print('deleting test data')
-    with arcpy.da.UpdateCursor(TABLE, ['OID@'], where_clause="FacilityName IN ('0', '1')") as cursor:
-        for _ in cursor:
-            cursor.deleteRow()
+    #arcpy.management.TruncateTable(TABLE)
```

Comment:
**chore**: we need to figure out the reset issue otherwise the subsequent tests will fail